src/createXcmTypes/ParaToSystem.ts

```diff
@@ -102,19 +103,22 @@ export const ParaToSystem: ICreateXcmType = {
 	 * @param amounts
 	 * @param xcmVersion
 	 */
-	createAssets: (
+	createAssets: async (
 		api: ApiPromise,
 		amounts: string[],
 		xcmVersion: number,
 		specName: string,
 		assets: string[],
-		opts: CreateAssetsOpts
-	): VersionedMultiAssets => {
-		const sortedAndDedupedMultiAssets = createParaToSystemMultiAssets(
+		opts: CreateAssetsOpts,
+		transferForeignAssets: boolean | undefined
```

Comment:
1. `transferForeignAssets?: boolean`

In addition to making this part of the options.